extensions/amp-video/1.0/video-iframe.js

```diff
@@ -193,7 +193,7 @@ export {VideoIframeInternal};
  * VideoWrapper using an <iframe> for implementation.
  * Usable on the AMP layer through VideoBaseElement.
  * @param {VideoIframeDef.Props} props
- * @param {{current: T|null}} ref
+ * @param {{current: (T|null)}} ref
```

Comment:
Not in this PR, but we're definitely gonna want an extern for `Ref<T>`, this type is everywhere in Bento
```suggestion
 * @param {{current: ?T}} ref
```